docs/api/combineManagers.md

```diff
@@ -0,0 +1,26 @@
+# `combineManagers(...managers)`
```

Comment:
I quite liked the `coordinator` suggestion. The only problem is that it's quite a long word for variable names